printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md

```diff
@@ -1,9 +1,9 @@
 # BTT Octopus vertical mount
 
 ## Overview
-This is a mount for new BTT octopus mount for DIN rail. It mounts the board vertically to take full advantage of the fan to cool those stepper motor drivers.
+This is a mount for new BTT Octopus mount for DIN rails. It mounts the board vertically to take full advantage of the fan to cool those stepper motor drivers.
 
-Remember to use the spacer and place it in between the board and mounts.
+Remember to use spacers and place it in between the board and mounts.
```

Comment:
Replace `it` with `them`.

Reply:
nice catch, thanks. English isn't my strong suit, lol.